stays/forms.py

```diff
@@ -0,0 +1,56 @@
+from .models import Stay, Review
+from django.forms import ModelForm
+from datetime import date
+
+
+class StayForm(ModelForm):
+    class Meta:
+        model = Stay
+        fields = ['owner', 'provider', 'start_date', 'end_date', 'pets']
+
+    def clean_start_date(self):
+        now = date.today()
+        data = self.cleaned_data['start_date']
+        if data and data > now:
+            self.add_error('start_date', 'Pick a valid date')
```

Comment:
The error message is pretty ambiguous

Reply:
Start date should not be from the past???